disnake/guild.py

```diff
@@ -4294,7 +4294,9 @@ async def get_or_fetch_members(
         if len(unresolved_ids) == 1:
             # fetch_member is cheaper than query_members
             try:
-                members.append(await self.fetch_member(unresolved_ids[0]))
+                member = await self.get_or_fetch_member(unresolved_ids[0])
+                if member is not None:
+                    members.append(member)
             except HTTPException:
                 pass
```

Comment:
This try-except-HTTPException is probably redundant, since you call `get_or_fetch_member` method with `strict=False`(defaut value) and HTTPException is silently ignored inside it

Reply:
Sorry, this PR was unfinished